src/utils/spacing/padding.css

```diff
@@ -0,0 +1,276 @@
+.u-p-xxsm {
```

Comment:
I commented in #49 as well, but I think we can do better on the variable names. Naming in our classes are `.xsm` and our vars are `--spacing-x-small`.

Can we figure out a single way of doing this?

Reply:
Yes! I created this before proposing the other scale. (I ran into the scale issues because of this :P )